Client/Frontend/Browser/BrowserViewController/BrowserViewController+WKNavigationDelegate.swift

```diff
@@ -316,9 +316,6 @@ extension BrowserViewController: WKNavigationDelegate {
 
         // Check if this response should be handed off to Passbook.
         if let passbookHelper = OpenPassBookHelper(request: request, response: response, canShowInWebView: canShowInWebView, forceDownload: forceDownload, browserViewController: self) {
-            // Clear the network activity indicator since our helper is handling the request.
-            UIApplication.shared.isNetworkActivityIndicatorVisible = false
```

Comment:
Last time i checked they are still used on iPad, do you want to remove them despite that?

Reply:
I think its fine we remove it, it's been deprecated with the scene based APIs and it doesn't make much sense anymore on iPad when you can have multiple windows of the same app (or even just multitasking apps). I think the loader we have on the URL bar should suffice